src/components/AssetEventsPanel/components/AssetEventsPanelPure.test.tsx

```diff
@@ -28,6 +30,42 @@ describe('AssetEventsPanel', () => {
       done();
     });
   });
+
+  it('should allow custom rendering of metadata', done => {
+    const assetEvents = fakeEvents;
+
+    const eventCssId = (event: CogniteEvent) => `pirate-${event.id}`;
+
+    const eventMessage = (event: CogniteEvent) => {
+      const source = (event.metadata && event.metadata.source) || '';
+      return `${source}: Look at me... Look at me! I'm the captain now!`;
```

Comment:
Can we have some `lorem ipsum` instead?)

Reply:
I have ipsum'ed the lorem!